src/loaders/common/loader_common_names.py

```diff
@@ -83,4 +83,11 @@
 MASH_METADATA = 'mash_run_metadata.json'
 
 # The fatal error file created if a data file cannot be successfully processed
-FATAL_ERROR_FILE = "fatal_error.json"
\ No newline at end of file
+FATAL_ERROR_FILE = "fatal_error.json"
+
+# key name for sample file in the metadata file for downloaded workspace objects
+SAMPLE_FILE_KEY = "sample_file"
+
+# extension for source sample data file and sample node data file for downloaded workspace objects
+SAMPLE_FILE_EXT = "sample"
+SAMPLE_NODE_FILE_EXT = "node_tree.sample"
```

Comment:
```suggestion
SAMPLE_PREPARED_EXT = "prepared.sample"
```

Or something like that, not really all that happy with "prepared". Just something that ways this version has be munged and is ready for the parser

Reply:
👍

Reply:
:+1: